Dockerfile

```diff
@@ -0,0 +1,346 @@
+RUN wget https://github.com/mikefarah/yq/releases/download/${YQ_VERSION}/yq_linux_amd64 -O /usr/bin/yq && chmod +x /usr/bin/yq
+ADD ddev-php-files /
+RUN apt-get -qq autoremove && apt-get -qq clean -y && rm -rf /var/lib/apt/lists/*
+RUN usermod -u ${WWW_UID} www-data && groupmod -g ${WWW_UID} www-data
+#END ddev-php-base
+
+### ---------------------------ddev-php-prod--------------------------------------
+### Build ddev-php-prod from ddev-php-base as a single layer
+### There aren't any differences
+FROM scratch AS ddev-php-prod
+COPY --from=ddev-php-base / /
+EXPOSE 8080 8585
+CMD ["/usr/sbin/php-fpm", "-F"]
+#END ddev-php-prod
+
+### ---------------------------nginx-base--------------------------------------
+### Build nginx-base
+FROM base as nginx-base
+ENV NGINX_FULL_VERSION=1.16.1-1~buster
+ENV NGINX_SHORT_VERSION=1.16.1
+RUN wget -q -O /tmp/nginx_signing.key http://nginx.org/keys/nginx_signing.key && \
+        apt-key add /tmp/nginx_signing.key
+RUN echo "deb http://nginx.org/packages/debian/ $(lsb_release -sc) nginx" >/etc/apt/sources.list.d/nginx.list
+RUN echo "deb-src http://nginx.org/packages/debian/ $(lsb_release -sc) nginx" >> /etc/apt/sources.list.d/nginx.list
+RUN apt-get -qq update
+RUN apt-get -qq install --no-install-recommends --no-install-suggests -y libcap2-bin nginx=${NGINX_FULL_VERSION}
+
+ADD nginx-base-files /
+
+RUN mkdir --parents \
+    /etc/nginx/sites-enabled \
+    /var/log/nginx \
+    /var/cache/nginx/client_temp \
+    /var/www/html \
+    && touch /var/log/nginx/access.log \
+    && touch /var/log/nginx/error.log \
+    && touch /etc/nginx/nginx.conf \
+    && chown -R www-data:www-data /var/www /var/cache/nginx /var/log/nginx /etc/nginx/sites-enabled /run \
+    && chmod -R 755 /var/www /etc/nginx/sites-enabled \
+    && chmod -R 766 /var/cache/nginx /var/log/nginx \
+    && chmod 744 /etc/nginx/nginx.conf
+
+EXPOSE 8080
+CMD ["nginx"]
+
+#END nginx-base
+
+### ---------------------------nginx-mod-builder--------------------------------------
+#nginx-mod-builder is a throwaway image just to build
+#needed nginx modules
+FROM nginx-base AS nginx-mod-builder
+ENV DEBIAN_FRONTEND noninteractive
+ENV ERRORS 0
+ENV PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
+ARG NGINX_OPENTRACING_VERSION=v0.9.0
+ARG VTS_VERSION=0.1.18
+
+RUN apt-get -qq --no-install-recommends --no-install-suggests -y install \
+    ncurses-bin \
+    build-essential \
+    cmake \
+    pkg-config \
+    libz-dev \
+    automake \
+    autogen \
+    autoconf  \
+    libtool
+RUN apt-get build-dep -y nginx=${NGINX_FULL_VERSION}
+WORKDIR /tmp
+### Grab opentracing
+RUN curl -sSL https://github.com/opentracing-contrib/nginx-opentracing/releases/download/${NGINX_OPENTRACING_VERSION}/linux-amd64-nginx-${NGINX_SHORT_VERSION}-ngx_http_module.so.tgz -o trace.tar.gz \
```

Comment:
Why do we build opentracing and vts nginx modules if they're not enabled in the nginx config? @jrthrawny (or anybody) do you remember? Should this just be removed? We could remove the nginx-mod-builder in that case. 

Note that the versions here were bumped to latest working (nginx 1.18.0 is not yet supported by those)

Reply:
I think these were enabled on the legacy hosting and during the process of transformation to new hosting, certain non-critical features may have been disabled or forgotten, sometimes just parts of those features.